CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 # Change Log
+## [v4.20.0](https://github.com/plivo/plivo-node/tree/v4.20.0) (2021-07-06)
+- Powerpack ID filter for list MDR
```

Comment:
this could be made more elaborate to cover the part that Powerpack ID has been added to the list & get MDR response, and Powerpack ID filter added for list MDR

Reply:
updated @neel-plivo